labkey-client-api/src/org/labkey/remoteapi/domain/ConditionalFormat.java

```diff
@@ -0,0 +1,110 @@
+package org.labkey.remoteapi.domain;
+
+import org.json.simple.JSONObject;
+
+public class ConditionalFormat
+{
+    private String queryFilter;
+    private String textColor = "";
+    private String backgroundColor = "";
+    private Boolean bold = false;
+    private Boolean italic = false;
+    private Boolean strikethrough = false;
+
+    public ConditionalFormat(String queryFilter, String textColor, String backgroundColor, Boolean bold, Boolean italic, Boolean strikethrough)
+    {
+        this.queryFilter = queryFilter;
```

Comment:
If you adopt the pattern of prepending class variables with underscore, then you can avoid having to qualify your class variables with : 'this'. For example : _queryFilter = queryFilter.